.github/actions/install-dependencies/action.yml

```diff
@@ -1,82 +1,56 @@
-name: install-dependencies
+name: Install Dependencies
 
 inputs:
-  dependency-identifiers:
-    type: string
+  msbuild-required-tools-target:
+    description: msbuild target to call to get a list of required external tools
+    default: GetRequiredExternalTools
+  required-tools:
+    description: external tools to install, in addition to the ones returned by `msbuild-required-tools-target`
   msbuild-dependency-target:
-    type: string
+    description: msbuild target to install dependencies
   solution-file-path:
-    type: string
+    description: path to the .sln file
   ckan-compatible-versions:
-    type: string
+    description: KSP versions for CKAN to be compatible with
     default: |
       1.12
       1.11
       1.10
       1.9
       1.8
-  ckan-filters:
-    type: string
-    default: |
-      .dds
-      .png
-      .bmp
-      .mu 
-      .mbm
-      .jpg
-      .wav
 
 runs:
   using: composite
   steps:
-    - name: ckan-install
-      shell: bash
-      run: |
-        sudo curl -sS -o /usr/share/keyrings/ksp-ckan-archive-keyring.gpg https://raw.githubusercontent.com/KSP-CKAN/CKAN/master/debian/ksp-ckan.gpg
-        echo 'deb [arch=amd64 signed-by=/usr/share/keyrings/ksp-ckan-archive-keyring.gpg] https://ksp-ckan.s3-us-west-2.amazonaws.com/deb stable main' | sudo tee /etc/apt/sources.list.d/ksp-ckan.list > /dev/null
-        sudo apt update
-        sudo apt install ckan
+    - name: Setup .NET
+      if: inputs.msbuild-dependency-target != '' && inputs.msbuild-required-tools-target != ''
```

Comment:
A little confused why this is conditional. is dotnet not ALWAYS a dependency? isnt installing it in 'install-dependencies' something you always want?

Reply:
I expect we’ll eventually be supporting mods that don’t compile code.